### ISSUE TITLE
Hide synced button does nothing?

### ISSUE THREAD

**Issue by mrmachine** [tegon/traktflix#136]
I think I have some 55 pages of history to review and sync. There are errors, so I do need to review them before I sync. It's too much to do in one sitting, so I'm trying to do a few pages at a time. But even after syncing a page of items, when I come back to the history page again it shows me all options again for review. The hide synced option doesn't do anything, as far as I can tell?

**Comment by rafaelgssa** [comment]
Are you syncing them with the release date?

**Comment by mrmachine** [comment]
@rafaelgssa yep, I have that option set as default in trakt itself and also enabled in the traktflix extension.

Does traktflix only compare the actual watch date from Netflix history with the watch date from Trakt, even when the option to add with release date is enabled? That seems like a bug in traktflix?

If traktflix is able to know and add with the release date, it should be able to compare against the release date?

**Comment by rafaelgssa** [comment]
Yep, there's currently a bug that doesn't detect synced items with release date. traktflix doesn't know the release date, it simply tells Trakt to add the item with the release date. I'll try to fix this soon, but I'm a bit out of time at the moment.

**Comment by Avamander** [comment]
This issue disappears once you refresh the page after initial sync.
